drivers/serial/uart_nrfx_uart.c

```diff
@@ -0,0 +1,443 @@
+/*
+ * Copyright (c) 2016-2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @brief Driver for Nordic Semiconductor nRF5X UART
+ */
+
+#include <uart.h>
+#include <gpio.h>
+#include <hal/nrf_uart.h>
+
+
+#ifdef CONFIG_UART_INTERRUPT_DRIVEN
+static uart_irq_callback_t m_irq_callback; /**< Callback function pointer */
+#endif /* CONFIG_UART_INTERRUPT_DRIVEN */
+
+/**
+ * @brief Set the baud rate
+ *
+ * This routine set the given baud rate for the UART.
+ *
+ * @param dev UART device struct
+ * @param baudrate Baud rate
+ * @param sys_clk_freq_hz System clock frequency in Hz
+ *
+ * @return N/A
+ */
+
+static int baudrate_set(struct device *dev)
+{
+	nrf_uart_baudrate_t nrf_baudrate; /* calculated baudrate divisor */
+
+	/* Use the common nRF5 macros */
```

Comment:
I don't really understand what the comment is telling me. But for sure, this "5" should be gone.